### DISCUSSION TITLE
Heroku Add-on

### DISCUSSION THREAD

**Discussion by ghost** [javascript-obfuscator/javascript-obfuscator#925] [General]
Hey! Would it be possible to make a Heroku add-on that obfuscates all .js files so the code that end users see is auto-obfuscated? Rather than manually obfuscate your files, then place these in a branch that gets uploaded to Heroku and keeping track of what files are what versions etc.

Doesn't have to only be Heroku. Anything that serves your code to the internet would be cool.

Anyways just a thought! I might try to work on it when I have a bit more time. But in the meantime, is there anything like this available?

**Comment by sanex3339** [comment]
Hi. I will be happy if someone creates obfuscator plugins for various platforms/bundlers/etc.
